lib/rmt/cli/repos.rb

```diff
@@ -16,6 +16,49 @@ def list
   end
   map 'ls' => :list
 
+  desc 'clean', _('Remove un-mirrored repositories from the local disk.')
+  def clean
+    base_directory = RMT::DEFAULT_MIRROR_DIR
+
+    repos_to_delete = Repository.where(mirroring_enabled: false).map do |repo|
+      repository_dir = File.join(base_directory, repo.local_path)
+      Dir.exist?(repository_dir) ? repo : nil
+    end.compact
+
+    if repos_to_delete.empty?
+      puts _('No un-mirrored repositories found on local disk.')
+      return
+    end
+
+    puts _('RMT found the following un-mirrored repositories:')
+    print "\n\e[31m"
+    repos_to_delete.each do |repo|
+      puts repo.description
+    end
+    print "\n\e[0m\e[1m"
+    print _('Would you like to continue and remove these directories from your local disk?')
+    print "\n\e[22m\s\s"
+    print _("Only '%{input}' will be accepted.") % { input: 'yes' }
+    print "\n\n\s\s\e[1m"
+    print _('Enter a value:')
+    print "\e[22m\s\s"
+
+    input = $stdin.gets.to_s.strip
+    if input != 'yes'
```

Comment:
I'm not sure if we have any guideline for user input on CLI tools, but what about having an option for a shorter, case-insensitive answer like only `y`:

```suggestion
    unless input.match?(/y(es)?/i)
```

If it's worth, we'd need to change line 41 above to include the shorter option.

On the other hand, accepting only `yes` is safer to the customer, two more keys to press before hitting enter could avoid accidentally accepting the clean command; `y` can lead to "automatic" behavior.

Reply:
I would probably just opt for the `yes` only option, as it is safer and not much more effort on the user's end.